recipes/farm/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "farm" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e7c7626ce80a4e532e0db3902c96630509c390e4f90a1a4f3402c496d565a79e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - setuptools
+    - wheel
```

Comment:
Kinda silly to have this as a run-time dependency, but that's what upstream does...